src/assets/json/checklist/tests-web-en.json

```diff
@@ -822,6 +822,20 @@
 	"wcag": ["1.3.5"],
 	"ID": "testWebID-55",
 	"IDorigin": "testWebID-55"
+}, {
+	"themes": "Forms",
+	"title": "Avoid redundant entries",
+	"type": ["Manual test"],
+	"tests": ["Identify processes that include filling out forms.", "Identify the fields requesting the same data."],
```

Comment:
Not sure about "Identify processes that include filling out forms".
Does-it means all processes including forms but not only? And what if a form is not part of multi-step process?

Reply:
proposal : https://github.com/Orange-OpenSource/a11y-guidelines/pull/591/commits/fa40ad7bc3e137d76a2b110d0f20c885ffb69078